docs/reference/slm/apis/slm-get.asciidoc

```diff
@@ -0,0 +1,125 @@
+[[slm-api-get-policy]]
+=== Get snapshot lifecycle policy API
+++++
+<titleabbrev>Get policy</titleabbrev>
+++++
+
+Retrieves one or more snapshot lifecycle policy definitions and
+information about the latest snapshot attempts.
+
+[[slm-api-get-request]]
+==== {api-request-title}
+
+`GET /_slm/policy/<policy-id>`
+
+`GET /_slm/policy/`
+
+[[slm-api-get-lifecycle-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the `manage_slm`
+cluster privilege to use this API. For more information, see
+<<security-privileges>>.
+
+[[slm-api-get-desc]]
+==== {api-description-title}
+
+Returns the specified policy definition and 
+information about the latest successful and failed attempts to create snapshots. 
+Includes the policy version and last modified date, 
```

Comment:
I don't think this is needed. If you decide to keep it, it's not a complete sentence as-is.
```suggestion
```